app/models.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Actor:
+    id: id
```

Comment:
The type annotation for `id` should be `int` instead of `id`. It should be `id: int`.